src/Functions/Public/properties-service/New-vRAPropertyGroup.ps1

```diff
@@ -0,0 +1,208 @@
+﻿function New-vRAPropertyGroup {
+    <#
+    .SYNOPSIS
+    Create a custom Property Group
+    
+    .DESCRIPTION
+    Create a custom Property Group
+
+    .PARAMETER Name
+    The unique name (ID) of the Property
+    
+    .PARAMETER Label
+    The text to display in forms for the Property
+
+    .PARAMETER Description
+    Description of the Property
+
+    .PARAMETER Tenant
+    The tenant in which to create the Property Group (Defaults to the connection tenant )
+
+    .PARAMETER Properties
+    The tenant in which to create the Property Group (Defaults to the connection tenant )
+   
+    .PARAMETER JSON
+    Property Group to send in JSON format
+
+    .INPUTS
+    System.String.
+
+    .OUTPUTS
+    System.Management.Automation.PSObject
+
+    .EXAMPLE
+    # Create a simple property group with no properties addded
+    New-vRAPropertyGroup -Name one 
+    
+    .EXAMPLE
+    # Create a property group with a description and label
+    New-vRAPropertyGroup -Name OneWithDescription -Label "On With Description" -Description "This is one with a label and description"
+
+    .EXAMPLE
+    # Create a property group with some properties added in simple form
+    New-vRAPropertyGroup -Name OneWithPropetiesSimple -Label "One With Properties" -Properties @{"com.org.bool"=$false; "com.org.string"="string1"}
+
+    .EXAMPLE
+    # Create a property group with some properties added in the extended form
+    New-vRAPropertyGroup -Name OneWithPropertiesExt -Label "One With Properties" -Properties @{"com.org.bool"=@{"mandatory"=$true; "defaultValue"=$false;}; "com.org.encryptedandshowonform"=@{"encrypted"=$true; "visibility"=$true; "defaultValue": "Un-encrypted string";};}
```

Comment:
Correct this example:
```"defaultValue": "Un-encrypted string"```

Should be this I think:

```"defaultValue"="Un-encrypted string"```

Reply:
Fixed and uploaded.